source/includes/admonitions/warn-configure-path.rst

```diff
@@ -1,19 +1,17 @@
-.. important::
 
-   To launch your selected MongoDB shell, you must:
+  To launch your selected MongoDB shell, you must:
```

Comment:
[nit] since we removed the `important::` admonition, we want to make sure the the text here has no indentations & is flushed to the margin of the page! 
```suggestion
To launch your selected MongoDB shell, you must:
```